.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+
+on:
+  workflow_dispatch:
+
+jobs:
+  linux-test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3.0.2
+
+    - name: Download artifacts
+      uses: actions/download-artifact@v3.0.0
+      with:
+        name: linux-numero
+        path: build/numero
+  
+    - name: Test
+      run: |
+        cd build/test
+        ctest
```

Comment:
I think it's assumed to be invoked as `make test`. This way it will ensure that all dependencies are built.

Reply:
Maybe. We'll see what it means on Windows as well. For now, I moved testing to the Build workflow as you suggested and it appears to work on Linux via Actions. Let me know if you can think of something else. Otherwise, let's merge and go from there towards other platforms and more tests :)

Reply:
Just add the same test step to all platforms - it should work out of the box or it will show where cross platform problems are.